css/selectors/selection.json

```diff
@@ -21,10 +21,16 @@
             "edge_mobile": {
               "version_added": true
             },
-            "firefox": {
-              "prefix": "-moz-",
-              "version_added": "1"
-            },
+            "firefox": [
+              {
+                "version_added": "62"
+              },
+              {
+                "version_added": "1",
+                "version_removed": "62",
```

Comment:
If I understand it correctly, -moz-selection is still around, so not removed in 62.

Reply:
Right, I missed that when looking [at the patch](https://bugzilla.mozilla.org/attachment.cgi?id=8962190&action=diff#a/servo/components/style/gecko/pseudo_element_definition.mako.rs_sec3):
```diff
@@ -234,6 +230,10 @@ impl PseudoElement {
                 return Some(${pseudo_element_variant(pseudo)})
             }
             % endfor
+            // Alias "-moz-selection" to "selection" at parse time.
+            "-moz-selection" => {
+                return Some(PseudoElement::Selection);
+            }
             _ => {
                 // FIXME: -moz-tree check should probably be
                 // ascii-case-insensitive.
```